native/src/routes/feedback/containers/FeedbackModalContainer.js

```diff
@@ -144,6 +144,8 @@ class FeedbackModalContainer extends React.Component<ContainerPropsType, Feedbac
     const label = t('contentOfCity', { city: cityName })
     const feedbackCategory = CONTENT_FEEDBACK_CATEGORY
     const feedbackType = route.params.type
+
+    // TODO: add 'POIS' in IGAPP-438
```

Comment:
So feedback is just possible for single pois now, right?

Reply:
No. It is possible for both. But feedback for pois overall cannot differed from General feedback from the Category Route.